### ISSUE TITLE
problem with gt06n 

### ISSUE THREAD

**Issue by yeddes** [tananaev/traccar#1730]
Hi . 
I want to run gt06n with traccar . traccar server is installed in my VPS but i i didn't receive anything from the device . 
```
STATUS#

Battery:4.13V,NORMAL; GPRS:Link Down; GSM Signal Level:Strong; GPS:Successful positioning, SVS Used in fix:11(11), GPS Signal Level:34,33,37,39,30,46,29,47,28,20,11; ACC:OFF; Defense:OFF
```
fichier log : 
```
2016-02-16 16:51:33  INFO: Operating System name: Linux version: 3.16.0-4-amd64$
2016-02-16 16:51:33  INFO: Java Runtime name: OpenJDK 64-Bit Server VM vendor: $
2016-02-16 16:51:33  INFO: Memory Limit heap: 643mb non-heap: 214mb
2016-02-16 16:51:33  INFO: Version: 3.1-SNAPSHOT
2016-02-16 16:51:35  INFO: Starting server...
```
thanks

**Comment by tananaev** [comment]
Well, there are two possibilities:

1. Your device doesn't send anything
2. Something blocks the connection (e.g. firewall)

**Comment by yeddes** [comment]
thanks . Can you help me to solve the problem ? 

GPRS:Link Down . does it mean there is no internet connection and block sending login and position message ?

**Comment by tananaev** [comment]
Yes, it sounds like internet doesn't work on the device, but it might be some other network issue. Not too sure.

**Comment by jamejo** [comment]
Two reasons are there for this.

1. Wrong APN Configuration (Find your APN for the SIM/Operator) and set it.
2. No data package/gprs package on your sim activated.

**Comment by yeddes** [comment]
thanks it was a data package problem . 
 new log file 

2016-02-18 19:34:19  INFO: Operating System name: Linux version: 3.16.0-4-amd64 architecture: amd64
2016-02-18 19:34:19  INFO: Java Runtime name: OpenJDK 64-Bit Server VM vendor: Oracle Corporation version: 24.95-b01
2016-02-18 19:34:19  INFO: Memory Limit heap: 643mb non-heap: 214mb
2016-02-18 19:34:19  INFO: Version: 3.1-SNAPSHOT
2016-02-18 19:34:20  INFO: Starting server...
2016-02-18 19:34:51  INFO: [475BEA6E] connected
2016-02-18 19:34:52 DEBUG: [475BEA6E: 5023 < 197.7.17.217] HEX: 7878110103587390503110432020320100006ab40d0a
2016-02-18 19:34:52  WARN: Unknown device - 358739050311043
2016-02-18 19:34:56 DEBUG: [475BEA6E: 5023 < 197.7.17.217] HEX: 78781101035873905031104320203201000249a60d0a
2016-02-18 19:34:56  WARN: Unknown device - 358739050311043
2016-02-18 19:34:58  INFO: [1C3252DB] connected
2016-02-18 19:35:06 DEBUG: [475BEA6E: 5023 < 197.7.17.217] HEX: 7878110103587390503110432020320100042c900d0a
2016-02-18 19:35:06  WARN: Unknown device - 358739050311043
2016-02-18 19:35:46  INFO: [475BEA6E] disconnected
2016-02-18 19:36:03  INFO: [B1AAAE4C] connected

but there is no position in the data base

**Comment by jamejo** [comment]
If '358739050311043' is your devices IMEI that your expecting positions from, you need to register the device under your account on traccar web.

Once you login on top left, there will be a + sign (Add device) give it a name, and in Identifier assign the '358739050311043' imei number of your device.

**Comment by yeddes** [comment]
thanks jamejo it works

**Comment by Jecri** [comment]
Hello, how did you solve it? I have a gps vt08s I configure my server, I have a server in node. but the status gprs is 'link down'
